src/components/back-link/template.test.js

```diff
@@ -0,0 +1,74 @@
+/* globals describe, it, expect */
+
+const { render, getExamples } = require('../../../lib/jest-helpers')
+
+const COMPONENT_NAME = 'back-link'
+
+const examples = getExamples(COMPONENT_NAME)
+
+describe('back-link component', () => {
+  it.skip('fails to render if the required fields are not included', () => {
+    // TODO: href is a required field but the component does not error,
+    // when it is not passed
+    expect(() => {
+      render(COMPONENT_NAME, {
+        text: 'Example'
+      })
+    }).toThrow()
+  })
+
+  it('renders the default with an anchor, href and text correctly', () => {
+    const { $ } = render(COMPONENT_NAME, examples.default)
+
+    const $component = $('.govuk-c-back-link')
+    expect($component.get(0).tagName).toEqual('a')
+    expect($component.attr('href')).toEqual('https://gov.uk')
+    expect($component.text()).toEqual('Back')
+  })
+
+  it('renders classes correctly', () => {
+    const { $ } = render(COMPONENT_NAME, {
+      classes: 'app-c-back-link--custom-class',
+      href: '#',
+      html: '<b>Back</b>'
+    })
+
+    const $component = $('.govuk-c-back-link')
+    expect($component.hasClass('app-c-back-link--custom-class')).toBeTruthy()
+  })
+
+  it('renders custom text correctly', () => {
+    const { $ } = render(COMPONENT_NAME, {
+      href: '#',
+      text: 'Home'
+    })
+
+    const $component = $('.govuk-c-back-link')
+    expect($component.text()).toEqual('Home')
```

Comment:
Can we have a test that ensures HTML is correctly escaped when passed as `text`?